IllegalStateException in UploadFilesActivity.onStop

GPlay:
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performStopActivityInner (ActivityThread.java:4805)
  at android.app.ActivityThread.handleStopActivity (ActivityThread.java:4858)
  at android.app.ActivityThread.-wrap25 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2028)
  at android.os.Handler.dispatchMessage (Handler.java:108)
  at android.os.Looper.loop (Looper.java:166)
  at android.app.ActivityThread.main (ActivityThread.java:7529)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:245)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:921)
Caused by: java.lang.IllegalStateException: 
  at androidx.fragment.app.FragmentManagerImpl.checkStateLoss (FragmentManager.java:2080)
  at androidx.fragment.app.FragmentManagerImpl.enqueueAction (FragmentManager.java:2106)
  at androidx.fragment.app.FragmentManagerImpl.popBackStack (FragmentManager.java:846)
  at androidx.fragment.app.DialogFragment.dismissInternal (DialogFragment.java:215)
  at androidx.fragment.app.DialogFragment.dismiss (DialogFragment.java:191)
  at com.owncloud.android.ui.activity.UploadFilesActivity.onStop (UploadFilesActivity.java:706)
  at android.app.Instrumentation.callActivityOnStop (Instrumentation.java:1386)
  at android.app.Activity.performStop (Activity.java:7587)
  at android.app.ActivityThread.performStopActivityInner (ActivityThread.java:4802)
```

--- a/src/main/java/com/owncloud/android/ui/activity/UploadFilesActivity.java
+++ b/src/main/java/com/owncloud/android/ui/activity/UploadFilesActivity.java
@@ -707,7 +707,7 @@ public class UploadFilesActivity extends FileActivity implements
     @Override
     protected void onStop() {
         if (dialog != null) {
-            dialog.dismiss();
+            dialog.dismissAllowingStateLoss();
         }
 
         super.onStop();

